src/fractalthorns_notifications.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (C) 2024 McAwesome (https://github.com/McAwesome123)
+# This script is licensed under the GNU Affero General Public License version 3 or later.
+# For more information, view the LICENSE file provided with this project
+# or visit: https://www.gnu.org/licenses/agpl-3.0.en.html
+
+# fractalthorns is a website created by Pierce Smith (https://github.com/pierce-smith1).
+# View it here: https://fractalthorns.com
+
+"""Code responsible for listening to and relaying SSE notifications."""
+
+import asyncio
+import aiohttp
+import json
+import logging
+import re
+from os import getenv
+from datetime import timedelta
+
+from aiohttp_sse_client2 import client as sse_client
+
+from src.fractalrhomb_globals import bot
+import src.fractalthorns_dataclasses as ftd
+
+notifs_logger = logging.getLogger("discord")
```

Comment:
I think ideally I want this to be something like `logging.getLogger("notifs")`, so they don't get mixed in with regular bot logging, but I couldn't figure out python logging and I figured asking you would be faster than researching it. So, uh, is there a way I can do that?

Reply:
As far as I can tell, `logging.getLogger` just gives you a logger with the specified name, creating it if it doesn't already exist.
`logging.getLogger("discord")` would give you the existing "discord" logger.
`logging.getLogger("notifs")` would give you a "notifs" logger that inherits from root.
`logging.getLogger("discord.notifs")` would give you a "notifs" logger that inherits from "discord".

When things like message format, (file) handlers, log levels, etc, it goes up the chain until it finds one. In this case, it would only matter for log level because that's the only thing I'm changing outside of the root logger, and I'm only changing it for the discord logger.

Reply:
The thing I couldn't figure out is that `logging.getLogger("notifs")` gives me a logger that doesn't do anything, none of the messages go anywhere, they don't go into discord.log and they don't create a notifs.log. I guess there's some switch I need to flip to actually get it to work.

Reply:
Was the log level set high enough to catch the logs?
Given that a lot of the logs are info, try running the bot with `-rv` and see if anything changes.

Reply:
Yep, you need -rv to see them.